src/radical/entk/appman/appmanager.py

```diff
@@ -22,6 +22,8 @@
 
 from .wfprocessor  import WFprocessor
 
+pika.connection.Parameters.DEFAULT_HEARTBEAT_INTERVAL = 0
+pika.connection.Parameters.DEFAULT_HEARTBEAT_TIMEOUT = 0
 
```

Comment:
Do we want that to be the default behavior for EnTK and RBMQ? Do we have issues only with one user and one machine or is the heartbeat_interval going to be a more general issue? If the former, then should we keep the default as it has been so far and enable setting the interval/timeout to a specific value via env variables?